package.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format2.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="2">
+  <name>urdfdom-headers</name>
+  <version>1.0.0</version>
+  <description>
+    C++ headers for URDF.
+  </description>
+  <maintainer email="steven@openrobotics.org">Steven! Ragnarök</maintainer>
+  <license>BSD</license>
+
+  <url type="website">http://ros.org/wiki/urdf</url>
+
+  <buildtool_depend>ament_cmake</buildtool_depend>
```

Comment:
I'm not sure if this is needed on a pure cmake package. We had something similar for catkin, but that was for the setup file. Since the dependency on the deb that provides the setup file is injected automatically by bloom now, I wonder if this package needs to depend on `ament_cmake` at all.

@dirk-thomas what do you think?